LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/JoinRequestMessageHandler.cs

```diff
@@ -55,7 +55,7 @@ async Task ProcessJoinRequestAsync(LoRaRequest request)
                 if (loRaDevice == null)
                 {
                     request.NotifyFailed(devEUI, LoRaDeviceRequestFailedReason.UnknownDevice);
-                    Logger.Log(devEUI, "join refused: unknown device", LogLevel.Information);
+                    Logger.Log(devEUI, "join refused", LogLevel.Information);
```

Comment:
Isn't that a duplicated log from LoRaDeviceRegistry line (289)?

Reply:
It is. I thought we talked about that. I find it difficult to ditch this as we would make an assumption about the implementation of the device registry component logging that particular one. We should not do that, since if the implementation changes, we do not get a log.
The one in the device registry is now more specific.